my-app/src/MapView.jsx

```diff
@@ -1,15 +1,78 @@
-import { GoogleMap, Marker, useLoadScript } from "@react-google-maps/api";
-import { useMemo } from "react";
+import {
+  GoogleMap,
+  InfoWindow,
+  Marker,
+  useLoadScript,
+} from "@react-google-maps/api";
+import { useEffect, useMemo, useState } from "react";
+
 import "./MapView.css";
 
 const MapView = () => {
   const { isLoaded } = useLoadScript({
     googleMapsApiKey: "AIzaSyCIdAofDyeBU8kc9VcVfBPGa30voIG7klc",
   });
-  const center = useMemo(() => ({ lat: 18.52043, lng: 73.856743 }), []);
-  <script async defer
-    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCIdAofDyeBU8kc9VcVfBPGa30voIG7klc&callback=initMap">
-  </script>
+  const center = useMemo(() => ({ lat: 37.4406279, lng: -122.1630952 }), []);
+  <script
+    async
+    defer
+    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCIdAofDyeBU8kc9VcVfBPGa30voIG7klc&callback=initMap"
+  ></script>;
```

Comment:
I'm a little confused about the placement of this script tag here. Since this is mixing regular HTML syntax with javascript, I would expect this to either be placed in an useEffect, or on the render function. 

Curious about what is the recommendation from google when integrating this api with react.